### PR TITLE
add recipe for package calcephpy

### PR DESCRIPTION
**Pull request by gastineau**
Would be good to have a package for [calcephpy](https://pypi.python.org/pypi/calcephpy) on conda forge

### CODE REVIEW

File: recipes/calcephpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: 'python setup.py install '
```
Comment (igortg) [inline recipes/calcephpy/meta.yaml]: Why the quotes? And can you use pip? ` python -m pip install --no-deps --ignore-installed .`

File: recipes/calcephpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+{% set hash_value = "e8b2e08306412355672ee945173bad3b67ce607558213945e061a86ce2af3107" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
```
Comment (igortg) [inline recipes/calcephpy/meta.yaml]: Using Jinja for parameter names is not common, and IMHO hurts readability

File: recipes/calcephpy/meta.yaml (hunk 1 above)
Comment (gastineau) [reply recipes/calcephpy/meta.yaml]: I use the command "conda skeleton pypi calceph" to generate the meta.yaml. 
So conda introduces these quotes in the generated script command.

File: recipes/calcephpy/meta.yaml (hunk 2 above)
Comment (gastineau) [reply recipes/calcephpy/meta.yaml]: In the same way, the command "conda skeleton pypi calceph" generates these custom command inside the meta.yaml. I prefer to keep that generated command if you are ok.

File: recipes/calcephpy/meta.yaml (hunk 1 above)
Comment (gastineau) [reply recipes/calcephpy/meta.yaml]: Any news for the merge request ?

File: recipes/calcephpy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python
+    - setuptools
+    - toolchain
```
Comment (xhochy) [inline recipes/calcephpy/meta.yaml]:
```suggestion
```

File: recipes/calcephpy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python
```
Comment (xhochy) [inline recipes/calcephpy/meta.yaml]:
```suggestion
  host:
    - python
    - pip
```

Comment (gastineau) [reply recipes/calcephpy/meta.yaml]: Thanks for the solution !

File: recipes/calcephpy/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    - setuptools
+    - cython
```
Comment (xhochy) [inline recipes/calcephpy/meta.yaml]: python, cython and numpy shouldn't be in the build, only in host. `build` is only for compilers/build-tools whereas `host` are the build-time dependencies.

**Pull request closed without merging**